testscript/testdata/wait.txt

```diff
@@ -1,6 +1,10 @@
 [!exec:echo] skip
 [!exec:false] skip
 
+# TODO: the '\n' below doesn't work on Windows on Github Actions, which does
+# have coreutils like "echo" installed. Perhaps they emit CRLF?
+[windows] skip
```

Comment:
How about dropping this reliance on `echo` altogether and defining a trivial `testscript.RunMain` command?

Reply:
I leave this for a future PR, hence the TODO :) I want 1.16 support fixed ASAP, really, and this feels like low priority in comparison. Travis did not have echo installed on windows, so this was skipped there anyway.